src/modules/client/components/planning/AuxiliaryIndicators.vue

```diff
@@ -50,7 +55,7 @@ export default {
       return (this.customersDetails.duration) / this.customersDetails.customerCount;
     },
     hoursToWork () {
-      return this.hoursDetails.diff && this.hoursDetails.diff.absencesHours
+      return get(this.hoursDetails.diff, 'absencesHours') && this.hoursDetails.counterAndDiffRelevant
```

Comment:
si tu utilises get tu peux mettre `get(this.hoursDetails, 'diff.absencesHours')` tant qu'a faire

Reply:
idem pour les autres

Reply:
Done !